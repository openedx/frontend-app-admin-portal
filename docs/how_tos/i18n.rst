####################
React App i18n HOWTO
####################

This document has moved to the frontend-platform repo: https://github.com/openedx/frontend-platform/blob/master/docs/how_tos/i18n.rst
