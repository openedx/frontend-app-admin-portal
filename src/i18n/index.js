import { messages as paragonMessages } from '@edx/paragon';

export default [
  paragonMessages,
];
