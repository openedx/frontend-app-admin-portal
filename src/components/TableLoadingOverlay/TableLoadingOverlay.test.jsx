import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from '@edx/frontend-platform/i18n';
import TableLoadingOverlay from '.';

describe('TableLoadingOverlay', () => {
  it('renders a loading overlay', () => {
    const tree = renderer
      .create((
        <IntlProvider locale="en">
          <TableLoadingOverlay />
        </IntlProvider>
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
