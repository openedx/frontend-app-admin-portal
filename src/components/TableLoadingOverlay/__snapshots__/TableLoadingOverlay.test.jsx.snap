// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableLoadingOverlay renders a loading overlay 1`] = `
Array [
  <div
    className="table-loading-overlay"
  />,
  <div
    className="table-loading-message d-flex align-items-center justify-content-center "
  >
    <div
      className="loading d-flex align-items-center justify-content-center loading"
    >
      Loading...
      <span
        className="sr-only"
      >
        Loading
      </span>
    </div>
  </div>,
]
`;
